src/features/virtualized-table-renderers/fileNameCellRenderer.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+import type { IntlShape } from 'react-intl';
+import { getFileExtension } from '../../utils/file';
+import messages from './messages';
+import FileIcon from '../../icons/file-icon';
+import FolderIcon from '../../icons/folder-icon';
+import Link from '../../components/link/Link';
+import baseCellRenderer from './baseCellRenderer';
+import type { FileNameCellRendererCellData, FileNameCellRendererParams } from './flowTypes';
+import './FileNameCell.scss';
+
+const fileNameCellRenderer = (intl: IntlShape) => (cellRendererParams: FileNameCellRendererParams) =>
+    baseCellRenderer(cellRendererParams, (cellValue: FileNameCellRendererCellData) => {
+        const { id, name, isExternal, type, onClick } = cellValue;
```

Comment:
If we want to have onClick, we should pass it back in as a param for the fileNameCellRenderer. CellValue should be kept as just the value.